libs/template/presets/storefront/experience/pages/registration-page.ts

```diff
@@ -0,0 +1,35 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
+import { featureVersion } from '@spryker-oryx/utilities';
+
+export const registrationPage: ExperienceComponent | undefined =
+  featureVersion === 'latest'
```

Comment:
```suggestion
  featureVersion >= '1.1'
```
It can be changed to `1.2`, depends on release schedule.

Reply:
Please make sure you add the `>=` condition